topics/icrc_34_delegation.md

```diff
@@ -136,38 +151,16 @@ the [IC interface specification, authentication section](https://internetcompute
 
 ## Message Processing
 
-1. The relying party sends a `icrc34_get_global_delegation` request to the signer.
+1. The relying party sends a `icrc34_delegation` request to the signer.
 2. Upon receiving the request, the signer validates whether it can process the message.
     * If the relying party has not been granted the permission to request the action, the signer sends a response with
       an error back to the relying party.
     * The relying party must make sure that the request complies with scope targets restriction.
-3. The signer tries to retrieve and verify the trusted origins according to the [ICRC-28](icrc_28_trusted_origins.md)
-   specification.
+3. If the request includes targets, the signer **MAY** try to retrieve and verify the trusted origins according to the [ICRC-28](icrc_28_trusted_origins.md) specification.
     * If the trusted origins cannot be retrieved for any of the given delegations targets or the relying party origin is
-      not within any of the trusted origin lists, the signer rejects the request and send a response with an error code
-      back to the relying party.
-4. The signer returns a signed delegation to the relying party
-
-```mermaid
-sequenceDiagram
```

Comment:
Why did you drop the sequence diagram? I think it should still be included here.

Reply:
Added